src/components/MenuAuctions/index.tsx

```diff
@@ -18,20 +18,19 @@ export const MenuAuctions: React.SFC<MenuAuctionProps> = ({
             <thead>
               <tr>
                 <th>Auction</th>
-                <th>Index</th>
                 <th>Committed</th>
                 <th>Claim Tokens</th>
               </tr>
             </thead>
             <tbody>
               {ongoingAuctions.map(
                 (auction, i) =>
-                  <tr key={`${auction.sell.address}-${auction.buy.address}-${i}`}>
+                  auction && <tr key={`${auction.sell.address}-${auction.buy.address}-${i}`}>
                     <td>{`${auction.sell.symbol}/${auction.buy.symbol}`}</td>
                     <td>{`${auction.indices[auction.indices.length - 1]}`}</td>
                     <td>
                       <p>{`${auction.balancePerIndex[auction.balancePerIndex.length - 1] || 'N/A'} ${auction.sell.symbol}`}</p>
-                      <p>{auction.balancePerIndexInverse.length > 0 && `${auction.balancePerIndexInverse[auction.balancePerIndexInverse.length - 1]} ${auction.buy.symbol}`}</p>
+                      <p>{`${auction.balancePerIndexInverse[auction.balancePerIndexInverse.length - 1] || 0} ${auction.buy.symbol}`}</p>
                     </td>
                     {auction.claim && <td><img src={require('assets/claim.svg')} /></td>}
```

Comment:
We should also show `claim` button for `claimInverse` but with a different `onClick` function.